Extensions/xkit_preferences.js

```diff
@@ -950,7 +950,9 @@ XKit.extensions.xkit_preferences = new Object({
 
 			for(var extension in mdata.extensions) {
 
-				m_html = m_html + XKit.extensions.xkit_preferences.gallery_parse_item(mdata.extensions[extension]);
+				if(mdata.extensions[extension].hidden != "true"){
```

Comment:
Why are we blacklisting xkit installer? It would never show up anyway
because it's and xkit internal extension...
On Sat, Oct 24, 2015 at 7:47 PM Wolvan <notifications@github.com> wrote:

> In Extensions/xkit_preferences.js
> <https://github.com/new-xkit/XKit/pull/666#discussion_r42939385>:
>
> > @@ -950,7 +950,9 @@ XKit.extensions.xkit_preferences = new Object({
> >
> >  			for(var extension in mdata.extensions) {
> >
> > -				m_html = m_html + XKit.extensions.xkit_preferences.gallery_parse_item(mdata.extensions[extension]);
> > +				if(mdata.extensions[extension].hidden != "true"){
>
> There is absolutely no reason to introduce a new attribute to hide an
> extension in the gallery.
> We have a blacklist you just add the extension to, see
> https://github.com/new-xkit/XKit/blob/master/Extensions/xkit_preferences.js#L1093
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/new-xkit/XKit/pull/666/files#r42939385>.
>